src/pages/Miruro/main.ts

```diff
@@ -57,14 +61,30 @@ export const Miruro: pageInterface = {
       j.$('.anime-title').parent().parent().before(j.html(selector));
     },
   },
+  overview: {
+    getTitle(url) {
+      return Miruro.sync.getTitle(url);
+    },
+    getIdentifier(url) {
+      return Miruro.sync.getIdentifier(url);
+    },
+    uiSelector(selector) {
+      j.$('.anime-title').parent().parent().parent().before(j.html(selector));
+    },
+    getMalUrl(provider) {
+      return Miruro.sync.getMalUrl!(provider);
+    },
+  },
   init(page) {
     api.storage.addStyle(
       require('!to-string-loader!css-loader!less-loader!./style.less').toString(),
     );
     let inte: NodeJS.Timer;
     utils.urlChangeDetect(() => ready());
-    j.$(document).ready(() => ready());
+    j.$(() => ready());
     function ready() {
+      if (!Miruro.isSyncPage(window.location.href) && !Miruro.isOverviewPage!(window.location.href))
+        return;
       page.reset();
```

Comment:
Please move the page.reset in front of the condition. This is necessary to remove malsync UI when opening an page that is not the sync or overview page.

Reply:
I have done the changes